src/tools/heat/UtilFinalizeHarvesterMutator.cs

```diff
@@ -856,7 +856,7 @@ private void MutateComponents()
                                         processed = true;
                                     }
                                 }
-                                else if (5 == parts.Length && String.Equals("win32", parts[4], StringComparison.OrdinalIgnoreCase))
+                                else if (5 == parts.Length && (String.Equals("win32", parts[4], StringComparison.OrdinalIgnoreCase) || String.Equals("win64", parts[4], StringComparison.OrdinalIgnoreCase)))
```

Comment:
Is 'arm64' also a potential option here?

Reply:
Seems like it could be an option here. However, it looks as if support for arm64 would require more changes throughout the whole repository. As Heat appears to be deprecated anyway it is maybe sufficient to just add support for x64 platforms here.

Reply:
Agreed, @Nico-Ko.